frontend/packages/dev-console/src/components/helm/helm-release-resources-utils.ts

```diff
@@ -0,0 +1,16 @@
+import * as _ from 'lodash';
+
+export const helmReleaseResourceKindFilter = (filters, resource) => {
+  if (!filters || !filters.selected || !filters.selected.size) {
+    return true;
+  }
+  return filters.selected.has(resource.kind);
+};
+
+export const flattenResources = (resources) => {
+  _.map(resources, (value, key) => {
+    _.forEach(value.data, (res) => (res.kind = key));
+  });
+  const result = _.flatMap(resources, 'data');
+  return result;
```

Comment:
This doesn't seem right here. You can probably use `Object.keys(resources).reduce` here. WDYT?